app/src/modelProviders/fine-tuned/getCompletion-2.ts

```diff
@@ -23,14 +28,10 @@ export async function getCompletion2(
 
     if (!model) throw new Error("No OpenAI model ID found");
 
-    const completion = await getOpenaiCompletion(fineTune.projectId, {
+    return getOpenaiCompletion(fineTune.projectId, {
       ...prunedInput,
       model,
     });
-    return {
```

Comment:
It was unclear to me why we're override the model on the way out here. Shouldn't it always be the same for fine tunes, and if it weren't wouldn't we want the one OpenAI provides?